index.bs

```diff
@@ -4900,6 +4995,57 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the [=public key credential source=] created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean credProps;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the credential properties as discovered by the client, if any. For example, a client is especially likely
+    to have knowledge about properties of credentials stored in [=platform authenticators=]. 
+    The client MAY omit any properties at its discretion, and SHOULD omit any properties that would be duplicated in the
+    authenticator extension output. This dictionary is initially defined with one credential property 
+    {{CredentialPropertiesOutput/rk}} to indicate if the authenticator has created or used a resident key 
+    credential. Other credential properties MAY be returned in this extension if client implementations wish to do so.
```

Comment:
> Other credential properties MAY be returned in this extension if client implementations wish to do so.

This seems to give clients carte blanche to add stuff to `CredentialPropertiesOutput`, which strikes me as odd for a W3C spec? Dunno.

Reply:
Me either - I'll remove it for now until there is a compelling reason to have more properties.